ddspipe_participants/src/cpp/participant/rtps/CommonParticipant.cpp

```diff
@@ -59,6 +59,7 @@ CommonParticipant::CommonParticipant(
     , payload_pool_(payload_pool)
     , discovery_database_(discovery_database)
     , domain_id_(domain_id)
+    , rtps_participant_(nullptr)
```

Comment:
Good catch! But could you please move initialization to the header? so it's the same as with the DDS one.